packages/match/src/main.jsx

```diff
@@ -0,0 +1,179 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import debug from 'debug';
+import CorrectAnswerToggle from '@pie-lib/correct-answer-toggle';
+import { Feedback } from '@pie-lib/render-ui';
+import AnswerGrid from './answer-grid';
+import { withStyles } from '@material-ui/core/styles';
+
+const log = debug('pie-ui:match:main');
+
+function shuffle(array) {
```

Comment:
fyi - shuffle is something that we'll need to address in the future at the controller level. the use case is that for a given session a shuffle should only happen once. with this being the case, it makes sense for the controller to shuffle and then persist the shuffled state. the ui will just use that shuffle if present on render.